overlord/configstate/configcore/services.go

```diff
@@ -34,6 +34,7 @@ import (
 var services = []struct{ configName, systemdName string }{
 	{"ssh", "ssh.service"},
 	{"rsyslog", "rsyslog.service"},
+	{"console-conf", "getty@*"},
```

Comment:
I think we need to restart both `getty@*` and `console-conf@*` to achieve the desired effect.

We have this in getty@.service:
```
# /usr/lib/systemd/system/getty@.service.d/console-conf.conf
[Service]
ExecStartPre=/bin/systemctl start console-conf@%i.service
```

And the console-conf@.service:
```
#  /lib/systemd/system/console-conf@.service
...
[Service]
Environment=PYTHONPATH=/usr/share/subiquity
ExecStartPre=/bin/systemctl stop getty@%I
ExecStart=/sbin/agetty -i -i -n --noclear -l /usr/share/subiquity/console-conf-wrapper %I $TERM
ExecStopPost=/bin/systemctl start getty@%I
```

If a console-conf@tty2 instance is already running, restating getty@tty2 will not stop it. From what I can tell it's because of the glob, when getty@<term> does not exist (i.e. isn't defined as a unit) it will not be restarted. In our images, there is a `getty@tty1` symlink in getty, so `systemctl list-units getty@*` lists only that service, and only that one gets restarted. However, when I start console-conf@tty2 manually,  snap set has no effect on it.

Reply:
maybe console-conf@*  is a better marker there now

```
{"console-conf", "console-conf@*"},
```